modules/hal_gigadevice/Kconfig

```diff
@@ -24,27 +24,29 @@ choice GD32_HXTAL_FREQUENCY
 	prompt "High speed external oscillator clock frequency"
 	default GD32_HXTAL_FIRMWARE_DEFINED if !SOC_SERIES_GD32VF103
 	default GD32_HXTAL_25MHZ if SOC_SERIES_GD32VF103
+	default GD32_HXTAL_8MHZ if SOC_SERIES_GD32E10X
 	help
-	  Define value of high speed crystal oscillator (HXTAL) in Hz
+	 Define value of high speed crystal oscillator (HXTAL) in Hz
```

Comment:
nit: space should not have been removed here

Reply:
fix here